pkg/restapi/kms/operation/models.go

```diff
@@ -15,6 +15,10 @@ type createKeyReq struct {
 	KeyType string `json:"keyType"`
 }
 
+type exportKeyResp struct {
+	PublicKey string `json:"publicKey"`
```

Comment:
do we wanna call it `PublicKey`, or `KeyBytes`

I called it `KeyBytes` in the framework [client](https://github.com/hyperledger/aries-framework-go/pull/2322/files#diff-37c4e3ce458fda264f50210b5839f93cca5a80f64c6eeb0650b4196378179cf5R64)

Reply:
we will use `PublicKey`